src/custom/components/swap/TradePrice/index.tsx

```diff
@@ -1,32 +1,9 @@
 import React from 'react'
 import styled from 'styled-components'
-import useLoadingWithTimeout from 'hooks/useLoadingWithTimeout'
-import { useIsQuoteRefreshing } from 'state/price/hooks'
 import TradePriceModComponent, { TradePriceProps } from './TradePriceMod'
-import { AnimatedImg } from 'pages/Swap'
-import loadingCowGif from 'assets/cow-swap/cow-load.gif'
-import repeat from 'assets/svg/repeat.svg'
-import { StyledBalanceMaxMini } from 'components/swap/styleds'
 
```

Comment:
We can likely skip the styled component all together at this point..

Reply:
just delete it 😂